packages/bff/src/services/catalogService.ts

```diff
@@ -13,11 +13,23 @@ import {
 import { BffAppContext, Headers } from "../utilities/context.js";
 import { getLatestAgreement } from "./agreementService.js";
 
```

Comment:
As a code style polishing and readability we can also use a const with state values?

```suggestion
const descriptorState = catalogApi.EServiceDescriptorState.Values;
```

Reply:
Yeah why not, we can also define them in a dedicated file so that we import them and use them in the same way around the BFF - done in [the last commit ](https://github.com/pagopa/interop-be-monorepo/pull/752/commits/76bb1a6ee84720cf99e3f5d94d9dc88f8639459c)